src/internet_identity_interface/src/internet_identity/types/api_v2.rs

```diff
@@ -73,6 +73,12 @@ pub enum AuthnMethodAddResponse {
     InvalidMetadata(String),
 }
 
+#[derive(Clone, Debug, CandidType, Deserialize, Eq, PartialEq)]
+pub enum AuthnMethodRemoveResponse {
```

Comment:
Would it make sense to return an error if the specified authn_method does not exist?

Reply:
It currently `traps` in that case. My idea was to first get a similar coverage on the API so that the front-ends could switch to v2 without losing functionality.
Then, we can start introducing additional error types. I think it makes more sense to do this holistically, because it affects a lot of other methods as well (i.e. the future `authn_method_replace`).

I'd like to split the work into smaller pieces and deal with errors later. Thankfully the APIv2 is designed to allow extending the API later in this way. 😉

Reply:
That makes sense, but maybe put a note about these plans in the code (e.g. a TODO) or in a JIRA ticket?

Reply:
I added it to the documentation in the `main.rs` file, because it applies to all the V2 API methods. 👍